modules/mono/mono_gd/gd_mono.h

```diff
@@ -59,6 +59,7 @@ struct PluginCallbacks {
 } // namespace gdmono
 
 class GDMono {
+	bool initialized;
```

Comment:
This needs to be initialized in the constructor or the crash this tries to fix happens anyways (at least in asan builds)

Tho I would maybe just move all the zeroing out of the constructor and into member initializers.